aspnetcore-features/CheckboxListSolution/Tests/Tests.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Likewise - .NET 7.

Reply:
Updated my VS to the latest version and .NET 7 is installed. Updated the project too to point to .NET 7.